lib/dotcom/system_status/subway.ex

```diff
@@ -129,6 +140,10 @@ defmodule Dotcom.SystemStatus.Subway do
   #
   # The exact implementation depends on which line. Green and Red have
   # branches, so they have special implementations.
+  @spec add_nested_statuses_for_line(String.t(), [Alert.t()], DateTime.t()) :: %{
```

Comment:
Should the first argument be `Routes.Route.id_t()`?

Reply:
Yes, probably 😇